tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -0,0 +1,126 @@
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/readonly/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1/testDir
+sudo umount $2
+
+# Run integration tests for rename_dir_limit directory with static mounting
+gcsfuse --rename-dir-limit=3 --implicit-dirs $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/rename_dir_limit/...  -p 1 --integrationTest -v --mountedDirectory=$2
+sudo umount $2
+
+gcsfuse --rename-dir-limit=3  $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/rename_dir_limit/...  -p 1 --integrationTest -v --mountedDirectory=$2
+sudo umount $2
+
+# Run integration tests for rename_dir_limit with --only-dr mounting.
+gcsfuse --only-dir testDir --rename-dir-limit=3 --implicit-dirs $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/rename_dir_limit/...  -p 1 --integrationTest -v --mountedDirectory=$2
+sudo umount $2
+
+gcsfuse --only-dir testDir --rename-dir-limit=3  $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/rename_dir_limit/...  -p 1 --integrationTest -v --mountedDirectory=$2
+sudo umount $2
+
+# Run integration tests for implicit_dir directory with static mounting
+gcsfuse --implicit-dirs $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1
+sudo umount $2
+
+gcsfuse --enable-storage-client-library=false --implicit-dirs  $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1
+sudo umount $2
+
+# Run integration tests for implicit_dir with --only-dr mounting.
+gcsfuse --only-dir testDir --implicit-dirs $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1/testDir
+sudo umount $2
+
+gcsfuse --only-dir testDir --enable-storage-client-library=false --implicit-dirs  $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1/testDir
+sudo umount $2
+
+# Run integration tests for explicit_dir directory with static mounting
+gcsfuse --enable-storage-client-library=true $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/explicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1
+sudo umount $2
+
+gcsfuse --enable-storage-client-library=false $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/explicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1
+sudo umount $2
+
+# Run integration tests for explicit_dir with --only-dr mounting.
+gcsfuse --only-dir testDir --enable-storage-client-library=true $1 $2
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/explicit_dir/...  -p 1 --integrationTest -v --mountedDirectory=$2 --testbucket=$1/testDir
```

Comment:
is there any reason that for some --only-dir mounting tests, we are passing --testbucket=$1 and for some --testbucket=$1/testDir ?

Reply:
Where?
If only-dir needs to pass --testbucket flag. it will be testbucket=$1/testDir only. 
without only-dir it is testbucket=$1

Reply:
My bad.